cmd/cli/plugin/feature/main.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"os"
+
+	"github.com/aunum/log"
+
+	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/command/plugin"
+	featurePlugin "github.com/vmware-tanzu/tanzu-framework/pkg/v1/sdk/features/plugin"
+)
+
+var descriptor = cliv1alpha1.PluginDescriptor{
+	Name:        "feature",
+	Description: "Operate on features and featuregates",
+	Version:     "v0.0.1",
```

Comment:
nit: version should probably be `v0.1.0` if it's a working plugin. I _think_ you can even remove this since plugins are versioned with the repo.

Reply:
thanks for the catch, changed this for local testing and somehow it slipped through.. fixed now